mmv1/third_party/terraform/services/cloudidentity/data_source_cloud_identity_groups.go.erb

```diff
@@ -84,7 +84,7 @@ func dataSourceGoogleCloudIdentityGroupsRead(d *schema.ResourceData, meta interf
 		return nil
 	})
 	if err != nil {
-		return transport_tpg.HandleNotFoundError(err, d, fmt.Sprintf("CloudIdentityGroups %q", d.Id()))
+		return transport_tpg.HandleDataSourceNotFoundError(err, d, fmt.Sprintf("CloudIdentityGroups %q", d.Id()), "")
```

Comment:
We don't have a usable ID or URL to send as the last parameter here, but what do you think about at least "Groups" or something?

Reply:
The reason nothing is passed is this error is never actually a 404 -- it returns a different web error, which means the argument is unused. That said, since I updated it to use the new function (was primarily meant to ensure the lack of breaking change should it need updating in the future), I'll update it to use "Groups" in case the 404 is hit in the future

Reply:
That makes sense, thanks for the explanation!